index.html

```diff
@@ -189,46 +189,130 @@
         Installable web applications
       </h2>
       <p>
-        This document attempts to address the <cite><a href=
-        "https://w3c-webmob.github.io/installable-webapps/">Use Cases and
-        Requirements for Installable Web Apps</a></cite> .
-      </p>
-      <p>
-        A web application is <dfn data-lt="installing">installed</dfn> if the
-        user agent has provided the end-user with a means of instantiating a
-        new <a>top-level browsing context</a> that has the manifest's members
-        applied to it. That is, the manifest's members, or their defaults, are
-        in effect on the <a>top-level browsing context</a> (see <a>application
-        context</a>).
+        A common use of a manifest is for a user agent to <dfn data-lt=
+        "installing|installed|installation">install</dfn> a web application:
+        whereby the user agent provides the end-user with a means of
+        instantiating a new <a>top-level browsing context</a> that has the
+        manifest's members applied to it. That is, the manifest's members, or
+        their defaults, are in effect on the <a>top-level browsing context</a>.
+        This distinguishes an installed web application from a traditional
+        bookmark, as opening a web page from a traditional bookmark will not
+        have a manifest <a>applied</a> to it.
       </p>
       <p>
-        An example of <a>installing</a> would be a user agent that allows the
-        end-user to add a shortcut to a web application on their device's
-        homescreen (using the name and one of the icons found in the manifest).
-        Then, when the end-user launches a web application through this
-        explicit means, the manifest is applied to the browsing context prior
-        to the <a>start URL</a> being loaded. This gives the user agent time to
-        apply the relevant values of the manifest, possibly changing the
-        display mode and screen orientation of the web application.
+        An example of a user agent that allows installation would be one that
+        provides the end-user with a means to add a shortcut to a web
+        application on their device's homescreen (using the name and one of the
```

Comment:
if could also well be to their launcher. Why not say that the application becomes part of the platforms normal way of launching applications, such as home screen, launcher, start menu etc